administrator/components/com_templates/View/Template/HtmlView.php

```diff
@@ -145,6 +145,7 @@ public function display($tpl = null)
 		$this->template    = $this->get('Template');
 		$this->preview     = $this->get('Preview');
 		$this->pluginState = PluginHelper::isEnabled('installer', 'override');
```

Comment:
Do we still need this variable?

Reply:
Yes, if plugin is disabled. So, nothing will update in database right ? Then, we don't show tab.

Reply:
Ok, thanks.